cmd/policy.go

```diff
@@ -0,0 +1,177 @@
+package cmd
+
+import (
+	"context"
+	"os"
+
+	"github.com/adityasaky/gittuf/internal/policy"
+	"github.com/adityasaky/gittuf/internal/repository"
+	"github.com/spf13/cobra"
+)
+
+var policyCmd = &cobra.Command{
+	Use:   "policy",
+	Short: "Tools to manage gittuf policies.",
+}
+
+var policyInitCmd = &cobra.Command{
+	Use:   "init",
+	Short: "Initialize policy file.",
+	RunE:  runPolicyInit,
+}
+
+var policyAddRuleCmd = &cobra.Command{
+	Use:   "add-rule",
+	Short: "Add a new rule to a policy file.",
+	RunE:  runPolicyAddRule,
+}
+
+var policyRemoveRuleCmd = &cobra.Command{
+	Use:   "remove-rule",
+	Short: "Remove rule from a policy file.",
+	RunE:  runPolicyRemoveRule,
+}
+
+var (
+	targetsFileName    string
+	ruleName           string
+	authorizedKeyFiles []string
+	rulePatterns       []string
+)
+
+// init for gittuf policy init
+func init() {
```

Comment:
I would generally limit the usage of init. While this works, Go init ordering is somewhat janky in that it relies on filename ordering, which can be a source of brittleness.

I'd recommend following gitsign/cosign here and having `New()` funcs that get built at main - https://github.com/sigstore/gitsign/blob/main/internal/commands/root/root.go

Having separate files that match the command structure also makes it easier to navigate to the source of each command.

Reply:
I've refactored `cmd` as a whole in the very last commit. Happy to submit it in a separate PR as it also modifies the `trust` subcommand and drops the `dev` one etc.